dbms/src/Storages/Transaction/TiKVRecordFormat.cpp

```diff
@@ -0,0 +1,176 @@
+// Copyright 2023 PingCAP, Ltd.
```

Comment:
this is a new file introduced by https://github.com/pingcap/tiflash/pull/7537. Should we also pick the changes into this PR to release-7.1?

Reply:
Yes